holoviews/plotting/widgets/__init__.py

```diff
@@ -108,7 +108,11 @@ def __init__(self, plot, renderer=None, **params):
         super(NdWidget, self).__init__(**params)
         self.id = plot.comm.target if plot.comm else uuid.uuid4().hex
         self.plot = plot
-        self.dimensions, self.keys = drop_streams(plot.streams,
```

Comment:
Ok...seems to be a result of ``plot.streams`` potentially having many more instances due to the nesting than before...